autoload/lsp/util.vim

```diff
@@ -154,7 +154,20 @@ enddef
 # number and character number. The user specified window command modifiers
 # (e.g. topleft) are in 'cmdmods'.
 export def JumpToLspLocation(location: dict<any>, cmdmods: string)
-  var fname = LspUriToFile(location.uri)
+  var uri: string
+  var range: dict<any>
+
+  if location->has_key('targetUri')
+    # LocationLink
+    uri = location.targetUri
+    range = location.targetSelectionRange
+  else
+    # Location
+    uri = location.uri
+    range = location.range
+  endif
```

Comment:
As this code is repeated a few times, can you move this to a function in the util file and use that?  This function will take a LSP location and return [uri, range]?

Reply:
In that case I think we should return three values: `targetUri, targetRange` and `targetSelectionRange`. Possible making `targetRange` empty when mapping over `Location`.

See https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#locationLink